boards/imix/chip_layout.ld

```diff
@@ -1,8 +1,8 @@
 /* Memory Spaces Definitions, 448K flash, 64K ram */
 ROM_ORIGIN  = 0x00010000; /* Use bootloader starting at 0x0000 */
-ROM_LENGTH  = 0x00020000;
-PROG_ORIGIN = 0x00030000;
-PROG_LENGTH = 0x00040000;
+ROM_LENGTH  = 0x00025000;
+PROG_ORIGIN = 0x00035000;
+PROG_LENGTH = 0x00035000;
```

Comment:
I think this 448K flash limit is a holder from storm's bootloader, and is already accounted for otherwise. We have 512kB total.

To simplify things, the layout should probably be:

```
ROM_ORIGIN  = 0x00010000;
ROM_LENGTH  = 0x00030000;
PROG_ORIGIN = 0x00040000;
PROG_LENGTH = 0x00040000;
```

Reply:
Got it, thanks.